pmd-ant/src/test/resources/net/sourceforge/pmd/ant/xml/cpdtasktest.xml

```diff
@@ -3,7 +3,7 @@
 	<property name="pmd.home" value="${user.dir}"/>
     <property name="src"      value="src/main/java"/>
 
-    <taskdef name="cpd" classname="net.sourceforge.pmd.cpd.CPDTask"/>
+    <taskdef name="cpd" classname="net.sourceforge.pmd.ant.CPDTask"/>
```

Comment:
This might be worth putting in the release notes as it is an incompatible change.

Reply:
Good point. There is also the CPD Task usage example in the documentation, that needs to be updated (https://pmd.github.io/latest/pmd_userdocs_cpd.html#ant-task)